packages/datetime/src/dateInput.tsx

```diff
@@ -98,6 +98,12 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
     outOfRangeMessage?: string;
 
     /**
+     * Whether the date popover should be inline.
+     * @default true
+     */
+    popoverInline?: boolean;
```

Comment:
@jchomicki understood. To be clear, I meant "That said, we may just want to wait for 2.0 **before we introduce a `popoverProps` prop.**" - meaning we could just move forward with the `popoverInline` for now if we're all cool with it. @giladgray Given that the dream is to support all popover props eventually, should we approve this PR to unblock @jchomicki in the short term?